src/app/organizations/organization-details/organization-details.component.html

```diff
@@ -1,10 +1,14 @@
+<div *ngIf="isOverview">
+    <app-header></app-header>
+</div>
```

Comment:
The header is needed to be added to the organization details screen for overview mode.